tests/snapshot.rs

```diff
@@ -46,6 +46,10 @@ fn test_build_graphs_and_create_embeddings() {
             in_memory_entity_mapping_persistor.clone(),
             &mut in_memory_embedding_persistor,
         );
+        in_memory_embedding_persistor
+            .entities
+            .sort_by_key(|e| e.entity.clone());
```

Comment:
FYI

```
51 |             .sort_by_key(|e| e.entity);
   |                              ^^^^^^^^ move occurs because `e.entity` has type `std::string::String`, which does not implement the `Copy` trait
```